roles/ceph-osd/tasks/start_osds.yml

```diff
@@ -0,0 +1,18 @@
+---
+- include: osd_fragment.yml
+  when: crush_location
+
+- name: get osd id
+  shell: |
+    ls /var/lib/ceph/osd/ | sed 's/.*-//'
+  changed_when: false
+  failed_when: false
+  always_run: true
+  register: osd_id
+
+- name: start and add that the osd service(s) to the init sequence
```

Comment:
I am not sure how this is going to work here. This would only work if ceph-volume has activated the OSD. The activation process is the one in charge of making sure that everything an OSD needs to start is there *including* starting the OSD process at the end.

In what situation one would want to start the OSD daemon like this?